include/ozo/impl/result.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <ozo/error.h>
+#include <ozo/type_traits.h>
+#include <ozo/concept.h>
+#include <ozo/impl/row.h>
+
+namespace ozo::impl {
+
+template <typename Rows, typename OidMap, typename Iterator>
+Require<Iterable<Rows> && (ForwardIterator<Iterator> || InsertIterator<Iterator>), error_code>
+convert_result(Rows&& rows, const OidMap& oid_map, Iterator result) {
```

Comment:
IMO it is better to use ```out``` name for output result to avoid using a part of function name as a parameter name.

Reply:
done